### PR TITLE
Adding the possibility to read vectors or matrices

### PR DESCRIPTION
**Pull request by loumalouomega**
I wasn't impossible to read any vector or matrix in the current implementation. For example for the LOCAL_INERTIA_TENSOR:

~~~json
            "Variables"        : {
                "THICKNESS"     : 1.0,
                "DENSITY"       : 7850.0,
                "YOUNG_MODULUS" : 206900000000.0,
                "POISSON_RATIO" : 0.29,
                "KratosMultiphysics.StructuralMechanicsApplication.CROSS_AREA"    : 1.0,
                "LOCAL_INERTIA_TENSOR"  : [[0.27,0.0],[0.0,0.27]]
            },
~~~

### CODE REVIEW

**Review by philbucher — APPROVED** [review]
I tested it, it works, thanks @loumalouomega

**Comment by RiccardoRossi** [comment]
can this be made to use the modifications in #713 ? (it could be done in two steps, first accepting the current input and then modifying it to use the GetMatrix GetVector methos in #713)

**Review by RiccardoRossi — CHANGES_REQUESTED** [review]
...however a test is missing for the funcitonality...

**Comment by philbucher** [comment]
Once #713 is merged this can be done a lot easier in fact, so maybe we should wait?

**Comment by loumalouomega** [comment]
@philbucher Do I remove this PR?

**Comment by philbucher** [comment]
@loumalouomega no please leave it. I will implement the necessary changed once #713 is merged

**Comment by loumalouomega** [comment]
OK

**Comment by philbucher** [comment]
I did the test @RiccardoRossi wanted.

Also I tried to change from json to Parameters, but it didn't work. There is a comment [here](https://github.com/KratosMultiphysics/Kratos/blob/feature-array-json-materials/kratos/python_scripts/read_materials_process.py#L41]:) "change to use the KratosParameters once dictionary iterators are exported"
Do you know if this has been done? @loumalouomega / @RiccardoRossi 

So basically only if json is changed to Parameters we can use the methods of #713

**Comment by philbucher** [comment]
Actually we can merge this now and do the change from json to parameters later in a separate pull request, if @RiccardoRossi is ok with this

File: kratos/python_scripts/read_materials_process.py [hunk 1]
```diff
@@ -86,7 +86,9 @@ def _AssignPropertyBlock(self, data):
                 },
                 "Variables" : {
                     "YOUNG_MODULUS" : 200e9,
-                    "POISSION_RATIO" : 0.3
+                    "POISSION_RATIO" : 0.3,
```
Comment (RiccardoRossi) [inline kratos/python_scripts/read_materials_process.py]: DOES THIS WORK? because it is badly spelt...

Comment (loumalouomega) [reply kratos/python_scripts/read_materials_process.py]: imagen

Comment (loumalouomega) [reply kratos/python_scripts/read_materials_process.py]: Los árboles no dejan ver el bosque

Comment (RiccardoRossi) [reply kratos/python_scripts/read_materials_process.py]: the big problem is that this should deliver an error. If it does not we have a problem

Comment (loumalouomega) [reply kratos/python_scripts/read_materials_process.py]: Is an example, is commented, not computed

Comment (philbucher) [reply kratos/python_scripts/read_materials_process.py]: It throws an error:
```
NameError: name 'POISSION_RATIO' is not defined
```

**Review by RiccardoRossi — APPROVED** [review]
